sentry-android-core/src/main/java/io/sentry/android/core/internal/util/SentryFrameMetricsCollector.java

```diff
@@ -94,7 +94,20 @@ public SentryFrameMetricsCollector(
     // https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:metrics/metrics-performance/src/main/java/androidx/metrics/performance/JankStatsApi24Impl.kt
 
     // The Choreographer instance must be accessed on the main thread
-    new Handler(Looper.getMainLooper()).post(() -> choreographer = Choreographer.getInstance());
+    new Handler(Looper.getMainLooper())
+        .post(
+            () -> {
+              try {
+                choreographer = Choreographer.getInstance();
```

Comment:
Since choregrapher is accessed via a handler outside the main thread I think it's a good idea to mark the field as `volatile` as well.

Reply:
I'm not sure, since it's only set once and read multiple times. We'd lose memory cache for that.
In the worst case now, when it's read it's still null, so the frame start will fallback to [now - frameDuration](https://github.com/getsentry/sentry-java/blob/bb0533f5d889d1aa7b81436d167fe06de3ca5935/sentry-android-core/src/main/java/io/sentry/android/core/internal/util/SentryFrameMetricsCollector.java#L118-L120)